packages/decent.js/docs/client.md

```diff
@@ -21,6 +21,9 @@ Attempts to log in using the provided credentials. Resolves with the newly logge
 ## await client.loginWithSessionID(sessionID: string)
 Attempts to log in using the provided sessionID. Resolves with the newly logged-in [user](user.md).
 
+## await client.logout(deleteSessionID: boolean)
+Logs out. `deleteSessionID` defaults to true.
+
```

Comment:
What is an example usage of not deleting the session ID?

Reply:
Say if you wanted to later login using the same session ID, I guess.

Reply:
Fair enough!